Index: packages/Gametech/Wallet/src/Http/Controllers/TopupController.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/packages/Gametech/Wallet/src/Http/Controllers/TopupController.php b/packages/Gametech/Wallet/src/Http/Controllers/TopupController.php
--- a/packages/Gametech/Wallet/src/Http/Controllers/TopupController.php	(revision 70d20566171815c348b92ee80e0fcc000c7f4125)
+++ b/packages/Gametech/Wallet/src/Http/Controllers/TopupController.php	(date 1618580012376)
@@ -64,17 +64,17 @@
         $bankss = collect($this->bankRepository->getBankInAccountAll()->toArray());
 
 
-        $banks = $bankss->transform(function ($item, $key) {
+        $bankss = $bankss->transform(function ($item, $key) {
             $item['filepic'] = Storage::url('bank_img/' . $item['filepic']);
             return $item;
         });
 
-//        if($profile->bank->shortcode === 'KBANK'){
-//            $banks = $bankss;
-//
-//        }else{
-//            $banks = $bankss->whereNotIn('shortcode',['KBANK'])->all();
-//        }
+        if($profile->bank->shortcode === 'KBANK'){
+            $banks = $bankss;
+
+        }else{
+            $banks = $bankss->whereNotIn('shortcode',['KBANK'])->all();
+        }
 
 
         return view($this->_config['view'], compact('banks', 'profile'));
